response_operations_ui/templates/reporting-unit.html

```diff
@@ -2,161 +2,178 @@
-                  {% endif %}
-                </td>
-              </tr>
-              {% endfor %}
-            </tbody>
-          </table>
-        </div>
+<dl class="item-headlines">
+  <dt class="item-headlines-term">
+    Reference:
+  </dt>
+  <dd id="RU_REF" class="item-headlines-definition">
+    {{ ru.sampleUnitRef }}
+  </dd>
+</dl>
+
+<h2 class="saturn">Surveys</h2>
+
+<section class="grid__col col-12@l ru-surveys">
+  {% for survey in surveys %}
+  <div class="ru-survey data-panel" id="survey-{{survey.shortName}}" name="associated-surveys">
+    <h3 name="survey-titles" class="saturn data-panel-header">
+      <button class='data-panel-header__title'></button>
+      {{ survey.surveyRef }} {{ survey.shortName }}
+    </h3>
 
-        <div class="ru-survey-respondents u-pt-s">
-          <h4 class="neptune">Respondents</h4>
+    <div class="data-panel-body">
+      <div class="ru-survey-ces">
 
-          <div class="ru-survey-enrolment-code u-pb-s" name="enrolment-code">
-            {% if survey.activeIacCode %}
-            Unused enrolment code:
-            <code class="mars" id="unused-enrolment-code-{{survey.shortName}}">{{survey.activeIacCode}}</code>
-            {% endif %}
-            <br/>
-            <a href="{{ url_for('reporting_unit_bp.generate_new_enrolment_code',
-                                collection_exercise_id=survey.collection_exercises[-1]['id'], ru_ref=ru.sampleUnitRef,
-                                trading_as=ru.name, survey_ref=survey.surveyRef, survey_name=survey.shortName) }}"
-               id="generate-new-code">
-              Generate new enrolment code
-            </a>
-          </div>
+        <h4 class="neptune">Collection exercises</h4>
 
-          <table name="tbl-respondents-for-survey" class="table table__dense tbl-ru-survey-respondents" summary="Respondents for {{survey.shortName}} for {{ ru.sampleUnitRef }}">
-            <thead class="table--head">
+        <table name="tbl-ce-for-survey" class="table table__dense" summary="Collection exercises for {{survey.shortName}} for {{ ru.sampleUnitRef }}">
+          <thead class="table--head u-vhx">
             <tr class="table--row">
               <th scope="col" class="table--header--cell">
-                Name and contact details
+                Period
               </th>
               <th scope="col" class="table--header--cell">
-                Enrolment status
+                Reporting unit name (from sample)
               </th>
               <th scope="col" class="table--header--cell">
-                Messages
+                Region (from sample)
+              </th>
+              <th scope="col" class="table--header--cell">
+                Status
               </th>
             </tr>
-            </thead>
-            <tbody>
-            {% for respondent in survey.respondents %}
+          </thead>
+          <tbody>
+            {% for ce in survey.collection_exercises[:4] %}
```

Comment:
Why are we only showing 4?

Reply:
This is in master not my code

Reply:
Oh didn't notice because of the reformatting

Reply:
Requirement is to only show 4 collection exercises.